clang/test/CodeGenSYCL/intel-fpga-loops.cpp

```diff
@@ -1,55 +1,35 @@
 // RUN: %clang_cc1 -triple spir64-unknown-unknown-sycldevice -disable-llvm-passes -fsycl-is-device -emit-llvm %s -o - | FileCheck %s
 
-// CHECK: br label %for.cond, !llvm.loop ![[MD_A:[0-9]+]]
-// CHECK: br label %for.cond, !llvm.loop ![[MD_B:[0-9]+]]
-// CHECK: br label %for.cond, !llvm.loop ![[MD_C:[0-9]+]]
-// CHECK: br label %for.cond2, !llvm.loop ![[MD_D:[0-9]+]]
-// CHECK: br label %for.cond,  !llvm.loop ![[MD_E:[0-9]+]]
-// CHECK: br label %for.cond2, !llvm.loop ![[MD_F:[0-9]+]]
+// CHECK: br label %for.cond,  !llvm.loop ![[MD_II:[0-9]+]]
```

Comment:
How about:
```suggestion
// CHECK: br label %for.cond,  !llvm.loop ![[MD_loop_II:[0-9]+]]
```
and similar?

Reply:
I don't really like this way, I prefer to have shorter filecheck labels here (lines 3-6)

Reply:
IMO, it's easy to get confused between `MD_II` and `MD_ii`, which, for some reason, mean different things.
Another suggestion would be to move lines 3-6 to precede/follow the corresponding loop bodies.

Reply:
From this point of view I agree. But to solve this confusion I'd rename `MD_ii` to `MD_ii_count`

Reply:
> Another suggestion would be to move lines 3-6 to precede/follow the corresponding loop bodies.

This one persists, yet feel free to disregard.

Reply:
@vmaksimo, this comment is still unresolved.
Are you going to update PR or it's ready for merge?

Reply:
@bader I believe it's ready for merge

Reply:
To make it clear, please, mark all comments as resolved.